### PR TITLE
MQE-917: Eliminate usage of XSD relative paths

### PR DESCRIPTION
**Pull request by KevinBKozan**
## This PR is a:
- [ ] New topic
- [x] Content fix or rewrite
- [ ] Bug fix or improvement
 
## Summary
MQE-917 two new mftf commands:
* `generate:urn-catalog` which mirrors Magento's command to help with auto completion in PHPStorm.
*  `upgrade:tests` which applies any MFTF major version upgrade scripts.

### CODE REVIEW

File: guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ vendor/bin/mftf command [options] [arguments]
 
 The following is a list of the most used commands.
 
+#### Options
+
+Option | Description|
+---|---
+`-u, --upgrade`   | Upgrades existing MFTF tests according to last major release requirements. Only tests in the default location will be upgraded with this flag.
```
Comment (dshevtsov) [inline guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md]: @KevinBKozan, is it the same as `upgrade:tests` below?

Comment (KevinBKozan) [reply guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md]: Yes, doing `build:project --upgrade` automatically runs the `upgrade:tests` command.

File: guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md [hunk 2]
```diff
@@ -240,6 +264,30 @@ Execute the `LoginCustomerTest.php` and `StorefrontCreateCustomerTest.php` tests
...
 
+### `upgrade:tests`
+
+Applies all MFTF major version upgrade scripts to test materials in given path (`test.xml`, `data.xml`, etc).
+
+#### Usage
+
+`upgrade:tests <path>`
+
+`<path>` is the path that contains MFTF test materials that need to be upgraded. The command searches subdirectories for any `*.xml` to attempt to upgrade.
```
Comment (dshevtsov) [inline guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md]: @KevinBKozan 
Is there a default path?
What if I run `upgrade:tests` with no path?

Comment (KevinBKozan) [reply guides/v2.2/magento-functional-testing-framework/release-2/commands/mftf.md]: There is no default path, path is required.

**Pull request merged**